build_web_compilers/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# 0.3.4
+# 0.3.4-dev
```

Comment:
We've generally feel this is not needed. One less thing to fix when you publish.

...but no big deal...

Reply:
hmm ok we have been adding this for the build packages at least